specs/trust-store-trust-policy.md

```diff
@@ -340,6 +510,10 @@ Notary Project allows user to execute custom validations during verification usi
 1. **Validate Revocation Status:**
     1. Validate signing identity(certificate and certificate chain) revocation status using [certificate revocation evaluation](#certificate-revocation-evaluation) section as per `signingIdentityRevocation` setting in trust-policy.
 1. Perform extended validation using the applicable (if any) plugin.
+1. Verify signature `payload`
+    1. Verify the signature envelope's `payload` matches the source [`payload`](./signature-specification.md#payload) that is getting verified.
```

Comment:
I think we should move this to step 3 "Validate Integrity" and add details about how the payload is constructed for each case, and which attributes are matched.